src/components/admin/members/MemberApprovalTable/MemberApprovalTableRow.tsx

```diff
@@ -56,36 +66,108 @@ export function MemberApprovalTableRow({
     approveUser({ userId, prepaidSessions });
   };
 
+  const name: string[] = (row.getValue("name") as string).split(" ");
+  const firstName = name.splice(0, name.length - 1).join(" ");
+  const lastName = name.splice(-1).join(" ");
+  const email: string = row.getValue("email");
```

Comment:
Can we make these assumptions? How about instead just using the firstName and lastName fields returned by the usePendingMembers hook